go/common/sgx/pcs/quote.go

```diff
@@ -100,7 +113,25 @@ func (q *Quote) Verify(ts time.Time, tcb *TCBBundle) (*TCBLevel, error) {
 		return nil, fmt.Errorf("pcs/quote: disallowed debug/production enclave/mode combination")
 	}
 
-	return q.Signature.Verify(&q.Header, &q.ISVReport, ts, tcb)
+	if policy == nil {
```

Comment:
Hm, is it ok that the hardcoded default here would likely require a consensus breaking change to update?

Reply:
The alternative could be that we would have this default as part of the registry parameters (this PR introduces the TEE features parameter) that could be more easily updated through upgrade handlers. And only if those parameters were not set would this default be used (or this could just return an error if the policy is `nil`).

Agree that this would be better from an upgradability standpoint, but it would add some complexity. Let me look into how that would look like.

Reply:
This has now been changed so that the default quote policy is part of the TEE features parameter.